record_platform_interface/lib/src/types/device.dart

```diff
@@ -0,0 +1,18 @@
+
+class Device {
+  int? id;
+  String? name;
+  String? channel;
```

Comment:
What's channel stands for? as String?

Reply:
```Channel:``` It gives you the information if it is mono or stereo, etc... At the same time it offers you the sampling frequency, it is the number of samples in time. The greater the number of samples, the better the audio quality.

Reply:
Ok, if fmedia provides those two pieces, then it should be separated.
`channel` should be an integer.